composables/getRemainingDays.test.ts

```diff
@@ -0,0 +1,33 @@
+import { getRemainingDaysInMonth } from './getRemainingDays'
+
+describe('getRemainingDaysInMonth', () => {
+
+    test("should return 30 if given the first day of the month in a 31 day month", () => {
+        // test case: january 1 2021
+        // Note: January is 0-based
+        const date = new Date(2021, 0, 1)
+        const result = getRemainingDaysInMonth(date)
+        expect(result).toBe(30)
+    })
+
```

Comment:
I like the idea of pure functions here

However... These tests still seem to be testing the API of `date` 

I think if I had created a composable like you did here though, during my tests and wrote some logic instead of just making the test pass, these tests would be more true to what I want to test